default_middlewares.go

```diff
@@ -77,6 +79,26 @@ func (rw *responseWriter) Write(b []byte) (int, error) {
 	return rw.ResponseWriter.Write(b)
 }
 
+func (rw *responseWriter) Flush() {
+	rw.ResponseWriter.(http.Flusher).Flush()
+}
```

Comment:
There is a problem for me here, especially because http.Flusher is not always present.

It could cause panic

```suggestion
func (rw *responseWriter) Flush() {
        flusher, ok := rw.ResponseWriter.(http.Flusher) 
         if ok { 
                 flusher.Flush()
         }
}
```

Reply:
I opened

- https://github.com/go-fuego/fuego/issues/439

Reply:
Thanks!